airbyte-integrations/connectors/source-pivotal-tracker/source_pivotal_tracker/manifest.yaml

```diff
@@ -0,0 +1,876 @@
+      retriever:
+        type: SimpleRetriever
+        requester:
+          $ref: "#/definitions/base_requester"
+          path: projects/{{ stream_slice['project_id'] }}/labels
+          http_method: GET
+        record_selector:
+          type: RecordSelector
+          extractor:
+            type: DpathExtractor
+            field_path: []
+        partition_router:
+          - type: SubstreamPartitionRouter
+            parent_stream_configs:
+              - type: ParentStreamConfig
+                parent_key: id
+                partition_field: project_id
+                stream:
+                  $ref: "#/definitions/streams/projects"
+      schema_loader:
+        type: InlineSchemaLoader
+        schema:
+          $ref: "#/schemas/labels"
+    releases:
+      type: DeclarativeStream
+      name: releases
+      primary_key:
+        - id
+      retriever:
+        type: SimpleRetriever
+        requester:
+          $ref: "#/definitions/base_requester"
+          path: projects/{{ stream_slice['project_id'] }}/releases
+          http_method: GET
+        record_selector:
+          type: RecordSelector
+          extractor:
+            type: DpathExtractor
+            field_path: []
+        partition_router:
+          - type: SubstreamPartitionRouter
+            parent_stream_configs:
+              - type: ParentStreamConfig
+                parent_key: id
+                partition_field: project_id
+                stream:
+                  $ref: "#/definitions/streams/projects"
+      schema_loader:
+        type: InlineSchemaLoader
+        schema:
+          $ref: "#/schemas/releases"
+    epics:
+      type: DeclarativeStream
+      name: epics
+      primary_key:
+        - id
+      retriever:
+        type: SimpleRetriever
+        requester:
+          $ref: "#/definitions/base_requester"
+          path: projects/{{ stream_slice['project_id'] }}/epics
+          http_method: GET
+        record_selector:
+          type: RecordSelector
+          extractor:
+            type: DpathExtractor
+            field_path: []
+        partition_router:
+          - type: SubstreamPartitionRouter
+            parent_stream_configs:
+              - type: ParentStreamConfig
+                parent_key: id
+                partition_field: project_id
+                stream:
+                  $ref: "#/definitions/streams/projects"
+      schema_loader:
+        type: InlineSchemaLoader
+        schema:
+          $ref: "#/schemas/epics"
+    activity:
+      type: DeclarativeStream
+      name: activity
+      primary_key:
+        - guid
+      retriever:
+        type: SimpleRetriever
+        requester:
+          $ref: "#/definitions/base_requester"
+          path: projects/{{ stream_slice['project_id'] }}/activity
+          http_method: GET
+        record_selector:
+          type: RecordSelector
+          extractor:
+            type: DpathExtractor
+            field_path: []
+        paginator:
```

Comment:
Please check how the pagination was implemented [originally](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/source-pivotal-tracker/source_pivotal_tracker/source.py#L22) and implement it in the same way for all streams here

Reply:
It is already implemented as standard pagination and verified its working

Reply:
Shouldn't these info be taken from headers?

Reply:
It finally comes down to standard offset pagination, I've verified it paginating in the builder itself and via test
![image](https://github.com/airbytehq/airbyte/assets/47392334/4e456dd5-f657-4945-aae7-3ced3c2039c0)